pkg/storage/secret/migrator/migrator.go

```diff
@@ -117,6 +118,21 @@ func initSecretStore(mg *migrator.Migrator) string {
 		Indices: []*migrator.Index{},
 	})
 
+	tables = append(tables, migrator.Table{
+		Name: TableNameSecureValueOutbox,
+		Columns: []*migrator.Column{
+			{Name: "uid", Type: migrator.DB_NVarchar, Length: 36, IsPrimaryKey: true}, // Fixed size of a UUID.
+			{Name: "message_type", Type: migrator.DB_TinyInt, Nullable: false},
+			{Name: "name", Type: migrator.DB_NVarchar, Length: 253, Nullable: false},      // Limit enforced by K8s.
+			{Name: "namespace", Type: migrator.DB_NVarchar, Length: 253, Nullable: false}, // Limit enforced by K8s.
+			{Name: "encrypted_secret", Type: migrator.DB_Blob, Nullable: false},
+			{Name: "keeper_name", Type: migrator.DB_Text, Nullable: false},
+			{Name: "external_id", Type: migrator.DB_NVarchar, Length: 36, Nullable: true}, // Fixed size of a UUID.
+			{Name: "created", Type: migrator.DB_BigInt, Nullable: false},
+		},
+		Indices: []*migrator.Index{},
```

Comment:
Could we benefit by having an index on `created` column?

Reply:
Just added that before seeing the comment 😆